docs/en/spark/configuration/source-plugins/neo4j.md

```diff
@@ -0,0 +1,137 @@
+
+for detail neo4j config message please visit [neo4j doc](#https://neo4j.com/docs/spark/current/reading/) 
+
+
+
+## Options
+
+| name                                                                     | type   | required | default value |
+| -------------------------------------------------------------------------| ------ | -------- | ------------- |
+| [result_table_name](#result.table.name-string)                           | string | yes      | -             |
+| [authentication.type](#authentication.type-string)                       | string | no       | -             |
+| [authentication.basic.username](#authentication.basic.username-string)   | string | no       | -             |
+| [authentication.basic.password](#authentication.basic.password-string)   | string | no       | -             |
+| [url](#url-string)                                                       | string | yes      | -             |
+| [query](#query-string)                                                   | string | yes*     | -             |
+| [labels](#labels-string)                                                 | string | yes*     | -             |
+| [relationship](#relationship-string)                                     | string | yes*     | -             |
+| [schema.flatten.limit](#schema.flatten.limit-string)                     | string | no       | -             |
+| [schema.strategy](#schema.strategy-string)                               | string | no       | -             |
+| [pushdown.filters.enabled](#pushdown.filters.enabled-string)             | string | no       | -             |
+| [pushdown.columns.enabled](#pushdown.columns.enabled-string)             | string | no       | -             |
+| [partitions](#partitions-string)                                         | string | no       | -             |
+| [query.count](#query.count-string)                                       | string | no       | -             |
+| [relationship.nodes.map](#relationship.nodes.map-string)                 | string | no       | -             |
+| [relationship.source.labels](#relationship.source.labels-string)         | string | Yes      | -             |
+| [relationship.target.labels](#relationship.target.labels-string)         | string | Yes      | -             |
+
+### result.table.name [string]
+
+result table name
+
+### authentication.type [string]
+
+authentication type
+
+### authentication.basic.username [string]
+
+username
+
+### authentication.basic.password [string]
+
+password
+
+### url [string]
+url 
+
+### query [string]
+
+Cypher query to read the data. you must specify  one way of (query labels relationship)
+
+### labels [string]
+
+List of node labels separated by : The first label will be the primary label. you must specify  one way of (query labels relationship)
+
+### relationship [string]
+
+Name of a relationship. you must specify  one way of (query labels relationship)
```

Comment:
You must specify one option from [query, labels OR relationship]

Just a suggestion :)